sai-challenge10/App.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TextInput,
+  ScrollView
+} from "react-native";
+import { NativeRouter, Route, Link } from "react-router-native";
+import Settings from "./Settings";
+import Profile from "./Profile";
+import Home from "./Home";
+import TimeStamp from "react-timestamp";
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      text: "",
+      text_list: [],
+      time_list: []
```

Comment:
are these for each message? would it make more sense to store an array of messages, where each message is an object with two properties, text and time